layouts/shortcodes/youtube.html

```diff
@@ -0,0 +1,8 @@
+<div class="responsive-video">
+  <iframe
+    src="https://www.youtube.com/embed/{{ index .Params 0 }}"
+    allowfullscreen
+    frameborder="0"
+  >
+  </iframe>
+</div>
```

Comment:
Any chance, we can get this closer to https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/shortcodes/youtube.html which is the current `youtube` shortcode we use?

Or can we maybe just use `{{< youtube id="abcabcba" "class"="responsive-video" >}}` now without introducing an override for the `youtube` shortcode?

Reply:
I think that what I did is a universal option, it will not require you to change anything in the documentation. In any case, it will be possible to resize or something else in the iframe

or make a copy with link and fix it for our needs?

Reply:
How about we add a new shortcode called `yt` that does this:

```
{{ $params := .Get 0 }}
{{ partial "youtube" (dict (class "responsive-video") (id $param1) }}
```

and move all occurrences to use it?

That way we would still benefit from everything that's defined in the hugo shortcode itself.

Reply:
ok

Reply:
Ok - I thought I had this working locally - unfortunately it doesn't.